llm/yamls/mosaic_gpt/350m.yaml

```diff
@@ -82,6 +89,7 @@ grad_clip_norm: 1.0
 
 # System
 seed: 17
+device_eval_batch_size: 8
 device_train_microbatch_size: 8
```

Comment:
If the 1b model support `device batch_size = 16`, then 350m model should also be set to at least 16.

Reply:
https://github.com/mosaicml/benchmarks/pull/26#discussion_r1029940651